Mage.Sets/src/mage/sets/exodus/Reconnaissance.java

```diff
@@ -0,0 +1,119 @@
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.sets.exodus;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.Rarity;
+import mage.constants.Zone;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.permanent.PermanentIdPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.Target;
+import mage.target.common.TargetControlledCreaturePermanent;
+
+/**
+ *
+ * @author spjspj
+ */
+public class Reconnaissance extends CardImpl {
+
+    public Reconnaissance(UUID ownerId) {
+        super(ownerId, 17, "Reconnaissance", Rarity.UNCOMMON, new CardType[]{CardType.ENCHANTMENT}, "{W}");
+        this.expansionSetCode = "EXO";
+
+        // {0}: Remove target attacking creature you control from combat and untap it.
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new ReconnaissanceRemoveFromCombatEffect(), new ManaCostsImpl("{0}"));
+        this.addAbility(ability);
+    }
+
+    public Reconnaissance(final Reconnaissance card) {
+        super(card);
+    }
+
+    @Override
+    public Reconnaissance copy() {
+        return new Reconnaissance(this);
+    }
+}
+
+class ReconnaissanceRemoveFromCombatEffect extends OneShotEffect {
+
+    public ReconnaissanceRemoveFromCombatEffect() {
+        super(Outcome.Benefit);
+        this.staticText = "Remove target attacking creature from combat and untap it";
+    }
+
+    public ReconnaissanceRemoveFromCombatEffect(final ReconnaissanceRemoveFromCombatEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public ReconnaissanceRemoveFromCombatEffect copy() {
+        return new ReconnaissanceRemoveFromCombatEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        List<UUID> attackers = game.getCombat().getAttackers();
+        Player player = game.getPlayer(source.getControllerId());
+
+        if (!attackers.isEmpty() && player != null) {
+            List<PermanentIdPredicate> uuidPredicates = new ArrayList<>();
+            for (UUID creatureId : attackers) {
+                uuidPredicates.add(new PermanentIdPredicate(creatureId));
+            }
+
+            FilterControlledCreaturePermanent filter = new FilterControlledCreaturePermanent("attacking creature controlled by you");
+            filter.add(Predicates.or(uuidPredicates));
+
+            Target target = new TargetControlledCreaturePermanent(0, 1, filter, false);
```

Comment:
Looks like target will be chosen on resolution, not on activation. Is it correct?